magento/pwa-studio
# [bugfix] Scaffolded projects failing on build

## Description

Updates the `@babel/preset-env` dependency on the Venia template project to prevent builds from failing because of the use of newer syntax. In this case, [optional chaining](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining).

## Related Issue







Closes PWA-2366

## Acceptance





Any developer

### Verification Stakeholders



Any developer

### Specification




## Verification Steps





1. Run the scaffolding command:

```shell
DEBUG_PROJECT_CREATION=1 yarn buildpack create-project scaffold-test --template "packages/venia-concept" --name "scaffold-test" --author "Scaffold Test" --backend-url "https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/" --npm-client "yarn"
```

2. Navigate to the new project: `cd scaffold-test`
3. Run the build command `yarn build`
4. Verify no build errors
5. Serve the application: `yarn start`
6. Verify application starts and functions correctly in the browser
7. Repeat steps **5** & **6** except with `yarn watch`

#### Test scenario(s) for direct fix/feature

1. Verify scaffolded app builds and runs





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
